hw_5/lib/screen/homepage.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/material.dart';
+import 'package:hw_5/helper/screen_helper.dart';
+import 'package:hw_5/model/Pref.dart';
+import 'package:hw_5/screen/interests.dart';
+
+class PersonalPage extends StatefulWidget {
+  @override
+  _PersonalPageState createState() => _PersonalPageState();
+}
+
+class _PersonalPageState extends State<PersonalPage> {
+  late final SharedPrefsService sharedPrefsService;
```

Comment:
This line causes an error, so use this instead 

`final SharedPrefsService sharedPrefsService = GetIt.I.get<SharedPrefsService>();`